proto/process/agent.proto

```diff
@@ -1108,6 +1113,14 @@ enum ResourceRequirementsType {
 	ephemeralContainer = 3;
 }
 
+enum ProtocolType {
+	protocolUnclassified = 0;
+	protocolUnknown = 1;
+	protocolHTTP = 2;
+	protocolHTTP2 = 3;
+	protocolTLS = 4;
+}
```

Comment:
It would be helpful to add that the backend needs to have mappings from these ids to metadata

Reply:
what do you mean by metadata?

Reply:
Like a mapping from protcolHTTP to the string "http"